src/tools/cephfs_mirror/PeerReplayer.cc

```diff
@@ -60,14 +60,12 @@ std::string snapshot_path(const std::string &snap_dir, const std::string &snap_n
   return snap_dir + "/" + snap_name;
 }
 
+auto entry_path = snapshot_path;
```

Comment:
No, snapshot_path uses `client_snapdir` which entry_path doesn't.

Reply:
I was mentioning about [this](https://github.com/ceph/ceph/blame/main/src/tools/cephfs_mirror/PeerReplayer.cc#L59-L61) overload of snapshot_path. This function and `entry_path` are the same.

Reply:
OK, I would still want to leave it as separate routines even if the implementation is same. This is so that its clear when invoking the helper as to what is being achieved and maybe the implementation of either of the two could change somewhere along.